akka-docs/src/test/java/jdocs/persistence/PersistenceMultiDocTest.java

```diff
@@ -4,28 +4,85 @@
 
 package jdocs.persistence;
 
+import akka.persistence.AbstractPersistentActor;
+import akka.persistence.RuntimePluginConfig;
 import akka.persistence.UntypedPersistentActor;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
 
 public class PersistenceMultiDocTest {
 
     //#default-plugins
-    abstract class ActorWithDefaultPlugins extends UntypedPersistentActor {
+    abstract class AbstractPersistentActorWithDefaultPlugins extends AbstractPersistentActor {
         @Override
-        public String persistenceId() { return "123"; }
+        public String persistenceId() {
+            return "123";
+        }
     }
     //#default-plugins
 
     //#override-plugins
-    abstract class ActorWithOverridePlugins extends UntypedPersistentActor {
+    abstract class AbstractPersistentActorWithOverridePlugins extends AbstractPersistentActor {
         @Override
-        public String persistenceId() { return "123"; }
+        public String persistenceId() {
+            return "123";
+        }
+
         // Absolute path to the journal plugin configuration entry in the `reference.conf`
         @Override
-        public String journalPluginId() { return "akka.persistence.chronicle.journal"; }
+        public String journalPluginId() {
+            return "akka.persistence.chronicle.journal";
+        }
+
         // Absolute path to the snapshot store plugin configuration entry in the `reference.conf`
         @Override
-        public String snapshotPluginId() { return "akka.persistence.chronicle.snapshot-store"; }
+        public String snapshotPluginId() {
+            return "akka.persistence.chronicle.snapshot-store";
+        }
     }
     //#override-plugins
 
+    //#runtime-config
+    abstract class AbstractPersistentActorWithRuntimePluginConfig extends AbstractPersistentActor implements RuntimePluginConfig {
```

Comment:
by making `RuntimePluginConfig` a pure interface (only abstract methods) this becomes more natural also (somewhat weird to `implements` a trait that is not a pure interface)